CI failure in integration tests due to:

```
2017-02-10 02:28:22 An exception occurred during task execution. The full traceback is:
2017-02-10 02:28:22 Traceback (most recent call last):
2017-02-10 02:28:22   File "/tmp/ansible_pEDMTz/ansible_module_setup.py", line 138, in <module>
2017-02-10 02:28:22     main()
2017-02-10 02:28:22   File "/tmp/ansible_pEDMTz/ansible_module_setup.py", line 128, in main
2017-02-10 02:28:22     supports_check_mode = True,
2017-02-10 02:28:22   File "/tmp/ansible_pEDMTz/ansible_modlib.zip/ansible/module_utils/basic.py", line 767, in __init__
2017-02-10 02:28:22   File "/tmp/ansible_pEDMTz/ansible_modlib.zip/ansible/module_utils/basic.py", line 1822, in _log_invocation
2017-02-10 02:28:22 NameError: global name 'arg_type' is not defined
```

--- a/lib/ansible/module_utils/basic.py
+++ b/lib/ansible/module_utils/basic.py
@@ -1819,7 +1819,7 @@ class AnsibleModule(object):
             if self.boolean(no_log):
                 log_args[param] = 'NOT_LOGGING_PARAMETER'
             # try to capture all passwords/passphrase named fields missed by no_log
-            elif arg_type != 'bool' and PASSWORD_MATCH.search(param) and \
+            elif PASSWORD_MATCH.search(param) and \
               arg_opts.get('type', 'str') != 'bool' and \
               not arg_opts.get('choices', False):
                 # skip boolean and enums as they are about 'password' state

